be/test/olap/rowset/segment_v2/bloom_filter_index_reader_writer_test.cpp

```diff
@@ -59,40 +59,46 @@ class BloomFilterIndexReaderWriterTest : public testing::Test {
 };
 
 template <FieldType type>
-void write_bloom_filter_index_file(const std::string& file_name, const void* values,
-                                   size_t value_count, size_t null_count,
-                                   ColumnIndexMetaPB* index_meta) {
+Status write_bloom_filter_index_file(const std::string& file_name, const void* values,
```

Comment:
use_primary_key_bloom_filter = true only in ut?

Reply:
write_bloom_filter_index_file is an UT function, when create primary key bf read/write test we set use_primary_key_bloom_filter true